gazebo_ros_control:
  pid_gains:
    joint_1:
      p: 10.0
      i: 1.0
      d: 0.0
    joint_2:
      p: 10.0
      i: 1.0
      d: 0.0
    joint_3:
      p: 10.0
      i: 1.0
      d: 0.0
    joint_4:
      p: 10.0
      i: 1.0
      d: 0.0

# Controller configs (if any)
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 500

joint_1_position_controller:
  type: position_controllers/JointPositionController
  joint: joint_1

joint_2_position_controller:
  type: position_controllers/JointPositionController
  joint: joint_2

joint_3_position_controller:
  type: position_controllers/JointPositionController
  joint: joint_3

joint_4_position_controller:
  type: position_controllers/JointPositionController
  joint: joint_4
